### PR TITLE
(EAI-235): Publish evaluation CLI package

### PR DESCRIPTION
**Pull request by mongodben**
Jira: https://jira.mongodb.org/browse/EAI-235

## Changes

- Add drone pipeline to publish eval CLI
- I also published eval package `0.1.0`

## Notes

-

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -871,3 +871,31 @@ steps:
...
+    - tag
+  ref:
+    include:
+      - refs/tags/mongodb-chatbot-evaluation-v*
+
+steps:
+  - name: build package
+    image: node:18
+    commands:
+      - npm install lerna
```
Comment (nlarew) [inline .drone.yml:891]:
```suggestion
      - npm install
```
We may add other dependencies (e.g. we just added `@stylistic/eslint-plugin-js`) should we default to installing everything?

**Review by nlarew — APPROVED** [review]
LGTM w/ one question

File: .drone.yml (hunk 1 above)
Comment (mongodben) [reply .drone.yml:891]: that's covered in the following `npm run bootstrap` step. it runs install

Comment (mongodben) [reply .drone.yml:891]: though if we upgrade lerna to latest, i believe they deprecated `bootstrap`, so we could refactor then.

**Pull request merged**